src/components/AppProvider/tests/AppProvider.test.tsx

```diff
@@ -1,57 +1,9 @@
 import * as React from 'react';
-import TestUtils from 'react-dom/test-utils';
 import {mount} from 'enzyme';
-import {createThemeContext} from '../../ThemeProvider';
-import {StickyManager, createAppProviderContext} from '../utilities';
 import {polarisAppProviderContextTypes} from '../types';
 import AppProvider from '../AppProvider';
 
 describe('<AppProvider />', () => {
-  it('passes i18n and withComponent properties to context', () => {
```

Comment:
This test will be updated and improved when we switch `AppProvider` to the new context API